Geometry/MTDGeometryBuilder/src/MTDParametersFromDD.cc

```diff
@@ -16,55 +18,32 @@ namespace {
       }
  
       int result(-1);
-      MTDStringToEnumParser<MTDTopologyMode::Mode> eparser;
-      MTDTopologyMode::Mode mode = (MTDTopologyMode::Mode) eparser.parseString(fvec[0]);
-      result = (int)(mode);
+      MTDTopologyMode::Mode eparser = MTDTopologyMode::MTDStringToEnumParser(fvec[0]);
+      result = (int)(eparser);
```

Comment:
If cast cannot be avoided, it would be better to use a named cast:
result = static_cast<int>(eparser);

These links have more info:
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es49-if-you-must-use-a-cast-use-a-named-cast
https://www.geeksforgeeks.org/static_cast-in-c-type-casting-operators/

Reply:
@cvuosalo thanks for the suggestion, I implement it assuming that the resi is ok for you and @ianna